LayerSelector.js

```diff
@@ -55,7 +55,7 @@ define([
         _getVisibleLayersAttr: function baseLayers() {
             var layers = this.get('managedLayers');
 
-            var visibleLayerWidgets = this.baseLayerWidgets.filter(function findVisibleBaseLayers(widget) {
+            var visibleLayerWidgets = array.filter(this.baseLayerWidgets, function findVisibleBaseLayers(widget) {
```

Comment:
Mainly the ones that we test against with the sauce. I don't want to pigeon hole versions but it is a good idea to users to know. 

I can see an app wanting to use the appliance layers with this tool and since it's not that complicated of a widget I figured it doesn't hurt to use some backward compat